RELEASE.md

```diff
@@ -107,6 +107,17 @@ See the [Change Log](https://github.com/kubeflow/pipelines/blob/master/CHANGELOG
 
 ## Cherry picking PRs to release branch
 
+### Option - cherrypick-approved label
```

Comment:
Is this manual or auto?

Reply:
which is manual or auto?
adding the label?

Reply:
Synced offline